engine/apps/email/tasks.py

```diff
@@ -65,7 +78,7 @@ def notify_user_async(user_pk, alert_group_pk, notification_policy_pk):
 
     emails_left = user.organization.emails_left(user)
     if emails_left <= 0:
-        UserNotificationPolicyLogRecord.objects.create(
+        _create_user_notification_policy_log_record(
```

Comment:
There are a few more UserNotificationPolicyLogRecord in this task, that need to be updated 👀

Reply:
good catch! addressed in https://github.com/grafana/oncall/pull/4635/commits/a70b1dfdf19bc11ae4cacd9ce42688867358e7c2